### DISCUSSION TITLE
questtion(Split): 多标签模式Split异常

### DISCUSSION THREAD

**Discussion by Kili33** [dotnetcore/BootstrapBlazor#4367] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

多标签模式：
![image](https://github.com/user-attachments/assets/f516cb17-174d-449a-bc44-5153ebc78fb5)
单页模式：
![image](https://github.com/user-attachments/assets/a6587b11-fa1a-4743-a3a1-01ead8722a5a)

### Expected Behavior

多标签模式正常使用

### Interactive render mode

Interactive Server (Interactive server-side rendering (interactive SSR) using Blazor Server)

### Steps To Reproduce

[BootstrapBlazorApp1.zip](https://github.com/user-attachments/files/17159956/BootstrapBlazorApp1.zip)

### .NET Version

8.0

**Comment by bb-auto** [comment]
@Kili33 Thank you for filing this issue. In order for us to investigate this issue, please provide [a minimalistic repro project](https://github.com/dotnetcore/bootstrapblazor/blob/main/docs/repro.md) that illustrates the problem.

**Comment by ArgoZhang** [comment]
@Kili33 父级容器给高度啊！！！
